LL/de_rigaib_muman_iar_cretim_inso.ttl

```diff
@@ -0,0 +1,580 @@
+@base <http://example.com/LL/de_rigaib_muman_iar_cretim_inso.ttl>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+@prefix rel: <http://purl.org/vocab/relationship/>.
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
+@prefix irishTitles: <http://example.com/earlyIrishTitles.ttl#> .
+
+<>
+    a dctype:Dataset;
+    dcterms:title "De Rigaib Muman Iar Cretim Inso"@sga;
+    dcterms:isFormatOf <http://www.ucc.ie/celt/published/G800011F/text006.html>;
+    dcterms:format "text/turtle".
+
+<#Oengus>
+    a irishTitles:Rí ;
+    irishTitles:reignLength 36 ;
+    rel:childOf <#NathFraich>;
+    irishTitles:succeededBy <#Eocho>;
+    foaf:name "Oengus" ;
+    owl:sameAs <http://example.com/LL/eognachta_casil.ttl#Oengusa>.
+
+<#NathFraich>
```

Comment:
Sets with just an owl:sameAs link have been causing problems for me when trying to review the data for missing links (they can't be sorted by name because there is no name). Can we start adding the foaf:name stuff? If so, no need to go back over the other instances - I'll sort any out as I go along.